src/nunjucks/moduk/components/input/__examples__/not-as-page-heading.njk

```diff
@@ -0,0 +1,9 @@
+{% from "moduk/components/input/macro.njk" import modukInput %}
+
+{{ modukInput({
+  label: {
+    text: "What is the name of the event?"
+  },
+  id: "event-name",
```

Comment:
We haven't been doing it so much to date but I was thinking it might be a good idea to keep the `id`s and `name`s unique in case we embed the examples directly on a page for the component the docs site.

Reply:
good point